src/connectors/hits-per-page/__tests__/connectHitsPerPage-test.ts

```diff
@@ -116,16 +130,19 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/hits-per-pa
 
     expect(renderFn).toHaveBeenCalledTimes(0);
 
-    const helper = algoliasearchHelper({}, '', {
+    const searchClient = createSearchClient();
+    const helper = algoliasearchHelper(searchClient, '', {
       hitsPerPage: 3,
     });
     helper.search = jest.fn();
 
-    widget.init({
-      helper,
-      state: helper.state,
-      createURL: () => '#',
-    });
+    widget.init!(
+      createInitOptions({
+        helper,
+        state: helper.state,
+        createURL: () => '#',
```

Comment:
You can remove the default `createURL` implems since they're handled in `createInitOptions`.